extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -347,6 +347,7 @@ export class AmpStoryPageAttachment extends AMP.BaseElement {
     this.state_ = AttachmentState.OPEN;
 
     this.storeService_.dispatch(Action.TOGGLE_SYSTEM_UI_IS_VISIBLE, false);
+    this.storeService_.dispatch(Action.TOGGLE_PAUSED, true);
```

Comment:
Either way is fine by me, but we should consider adding a `PageAttachment` StateProperty in the store that handles these actions.

Reply:
My idea was to wait until another component relies on the ATTACHMENT_STATE before adding a new action in the store service, but it'd be cool to dispatch one action instead of two...